types/helpers/helpers.canvas.d.ts

```diff
@@ -89,6 +89,38 @@ export interface RenderTextOpts {
    * Underline the text
    */
   underline?: boolean;
+
+  /**
+   * Dimensions for drawing the label backdrop
+   */
+  backdrop?: BackdropDimensions;
+}
+
+export interface BackdropDimensions {
```

Comment:
```suggestion
  backdrop?: Backdrop;
}

export interface Backdrop {
```

it is more as dimensions alone

Reply:
Renamed to BackdropOptions